importexport/guitarpro/importgtp-gp5.cpp

```diff
@@ -1437,6 +1437,8 @@ bool GuitarPro5::readNote(int string, Note* note)
             note->setHeadGroup(NoteHead::Group::HEAD_CROSS);
             note->setGhost(true);
             }
+      if (fretNumber > 0 && string == 4)
+          fretNumber = staff->part()->instrument()->stringData()->adjustBanjo5thFret(fretNumber);
```

Comment:
How does this work on any plucked string instrument with more than 4 strings that is not a Banjo?

Reply:
adjustBanjo5thFret() makes sure number of strings == 5 and uses 'startFret' which is only set for 5-string banjos.